snuba/datasets/storages/factory.py

```diff
@@ -33,15 +34,27 @@ def __init__(self) -> None:
             self.__initialize()
 
     def __initialize(self) -> None:
-        self._config_built_storages = {
-            storage.get_storage_key(): storage
-            for storage in [
-                build_storage_from_config(config_file)
-                for config_file in glob(
-                    settings.STORAGE_CONFIG_FILES_GLOB, recursive=True
-                )
-            ]
-        }
+        if get_config("load_configs_multi_processed", 0):
+            with Pool() as p:
```

Comment:
By default, `Pool()` uses all available cores, should I specify an actual number of max processes (cores)?